src/universe/analytics.ts

```diff
@@ -28,6 +30,14 @@ export interface UniverseAnalyticsEventsOptions {
   datepart?: string
 }
 
+export interface UniverseRevenueMetricsOptions {
+  groupBy?: 'day' | 'week' | 'month' | 'year'
+  version: 'last_touch' | 'testing'
+  start: Date
+  end: Date
+  fields?: [keyof RevenueMetricsValue]
```

Comment:
todo: mentioned in the type definition, we should use a new interface that only contains the optional fields :)